src/main/java/mekanism/common/integration/multipart/MultipartMekanism.java

```diff
@@ -162,4 +165,23 @@ public static boolean hasConnectionWith(TileEntity tile, EnumFacing side)
 		
 		return Collections.emptyList();
 	}
+	
+	public static boolean isMultiPartContainer(TileEntity entity)
```

Comment:
Correct. This was created simply to keep direct references to the `mcmultipart.api` packages restricted to the `mekanism.common.integration.multipart` package.